workitem/number_sequence/workitem_number_sequence_repository.go

```diff
@@ -0,0 +1,56 @@
+package numbersequence
+
+import (
+	"context"
+
+	"github.com/fabric8-services/fabric8-wit/log"
+	"github.com/jinzhu/gorm"
+	errs "github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+)
+
+// WorkItemNumberSequenceRepository the interface for the work item number sequence repository
+type WorkItemNumberSequenceRepository interface {
+	Create(ctx context.Context, spaceID uuid.UUID) (*WorkItemNumberSequence, error)
+	NextVal(ctx context.Context, spaceID uuid.UUID) (*WorkItemNumberSequence, error)
```

Comment:
Why return WorkItemNumberSequence and not just the Number?

Reply:
good point, indeed